platemate/food/views.py

```diff
@@ -136,6 +143,29 @@ def fe_day(request, day):
     except ValueError:
         return HttpResponseNotFound("Invalid date")
 
+@csrf_exempt
+def api_submission_statuses(request):
+    if request.method == 'POST':
+        try:
+            import pdb; pdb.set_trace()
+            json_data = json.loads(request.body)
+            submission_ids = json_data['ids']
+            response_json = {}
+            for submission_id in submission_ids:
+                status = 'NOT_FOUND'
+                data = {}
+                matching_submissions = Submission.objects.filter(id = submission_id)
```

Comment:
Should you be using `get` here?

http://stackoverflow.com/questions/4300365/django-database-query-how-to-get-object-by-id